### PR TITLE
ArduPlane: added RC option for LOITER mode

### PR DESCRIPTION
**Pull request by giacomo892**
Handy to have, and much better than use CIRCLE to keep the plane waiting for new inputs.

Thanks @shellixyz for the hint.

### CODE REVIEW

**Comment by IamPete1** [comment]
Great!, can you add it to the param description too please. https://github.com/ArduPilot/ardupilot/blob/fba5732292a3b95dbd02b817a2ccfc5302f45971/libraries/RC_Channel/RC_Channel.cpp#L94

Loiter would be 56 https://github.com/ArduPilot/ardupilot/blob/fba5732292a3b95dbd02b817a2ccfc5302f45971/libraries/RC_Channel/RC_Channel.h#L156

**Comment by giacomo892** [comment]
@IamPete1 sure, done!

**Comment by kd0aij** [comment]
I think you also need to handle the new option in init_aux_function (same as for CIRCLE)

File: libraries/RC_Channel/RC_Channel.cpp [hunk 1]
```diff
@@ -91,7 +91,7 @@ const AP_Param::GroupInfo RC_Channel::var_info[] = {
...
-    // @Values{Plane}: 0:Do Nothing, 4:ModeRTL, 9:Camera Trigger, 16:ModeAuto, 24:Auto Mission Reset, 28:Relay On/Off, 29:Landing Gear, 34:Relay2 On/Off, 30:Lost Plane Sound, 31:Motor Emergency Stop, 35:Relay3 On/Off, 36:Relay4 On/Off, 38:ADSB Avoidance En, 41:ArmDisarm, 43:InvertedFlight, 46:RC Override Enable, 51:ModeManual, 55:ModeGuided, 58:Clear Waypoints, 62:Compass Learn, 64:Reverse Throttle, 65:GPS Disable, 66:Relay5, 67:Relay6, 72:ModeCircle, 77:ModeTakeoff, 100:KillIMU1, 101:KillIMU2
+    // @Values{Plane}: 0:Do Nothing, 4:ModeRTL, 9:Camera Trigger, 16:ModeAuto, 24:Auto Mission Reset, 28:Relay On/Off, 29:Landing Gear, 34:Relay2 On/Off, 30:Lost Plane Sound, 31:Motor Emergency Stop, 35:Relay3 On/Off, 36:Relay4 On/Off, 38:ADSB Avoidance En, 41:ArmDisarm, 43:InvertedFlight, 46:RC Override Enable, 51:ModeManual, 55:ModeGuided, 58:Clear Waypoints, 6:Compass Learn, 64:Reverse Throttle, 65:GPS Disable, 66:Relay5, 67:Relay6, 72:ModeCircle, 56:ModeLoiter, 77:ModeTakeoff, 100:KillIMU1, 101:KillIMU2
```
Comment (Hwurzburg) [inline libraries/RC_Channel/RC_Channel.cpp]: mistake on compass learn....keep options in numerical order

**Comment by giacomo892** [comment]
Should be ready to merge. Can someone kindly take a look?

**Comment by tridge** [comment]
merged, thanks!

**Pull request closed without merging**